cmd/ddev/cmd/list_test.go

```diff
@@ -1,9 +1,12 @@
 package cmd
 
 import (
+	"bytes"
 	"testing"
 
 	"encoding/json"
+	oexec "os/exec"
```

Comment:
Just curious... why the alias here?

Reply:
Because another exec package was already imported.

Reply:
We probably shouldn't have named our package "exec" :(